phpScripts/editAboutMe.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+
+	/**
+	 * This function retrieves a single paragraph based on the given id.
+	 *
+	 * @param PDO $db accepts a valid database connection
+	 * @param int $id accepts the id of a paragraph
+	 *
+	 * @return array of a paragraph
```

Comment:
I'd possibly challenge that 'array of a paragraph' isn't very descriptive here? What should I expect to be in the array, etc?